VCFv4.3.tex

```diff
@@ -1277,7 +1288,7 @@ \subsection{Overall file organization}
 No record may refer to a contig not present in the header itself.
 
 \item All INFO and GENOTYPE fields must be fully typed in the BCF2 header to
-enable type-specific encoding of the fields in records.  An error should be
+enable type-specific encoding of the fields in records.  An error must be
```

Comment:
If the field is defined by the standard, an implementation could potentially perform the conversion by interpreting the field using the standard definition. I don't think we should ban implementations from attempting this by making them throw an error.